File: C/max_consecutive_ones.c
int findMaxConsecutiveOnes(int* nums, int numsSize)
{
    int i;
    int count;
    int max;
    
    i = 0;
    count = 0;
    max = 0;
    while (i < numsSize)
    {
        if (nums[i] == 1)
        {
            count++;
            max = count > max ? count : max;
        }
        if (nums[i] == 0)
            count = 0;
        i++;
    }
    return (max);
}

File: Python3/max_consecutive_ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_value, return_value = 0,0
        for num in nums:
            if num == 1 :
                max_value += 1
                if max_value > return_value:
                    return_value = max_value
            else:
                max_value = 0
        return return_value
        
